### ISSUE TITLE
Install from source instructions

### ISSUE THREAD

**Issue by kees-closed** [memetrooper/TEXTREME#10]
Hi,

I would like to package this into an RPM (when I have time, in a few months). Some documentation is missing about installing this from source. If you add these instructions such as dependencies and build steps to README.md, then I might create an RPM soonish for Fedora.

**Comment by benjhar** [comment]
I don't believe it is possible to compile to RPM in Godot anyway, but you can compile to .86 and .86_64

**Comment by kees-closed** [comment]
@thenamesweretakenalready that makes no sense :) RPM is just a way to distribute software, it's not an executable. With dependency and build information I can create an RPM to distribute it on e.g. Fedora systems.

**Comment by maksloboda** [comment]
All you can with the project in godot is exporting it, if you want to compile godot check the docs
http://docs.godotengine.org/en/3.1/development/compiling/index.html
